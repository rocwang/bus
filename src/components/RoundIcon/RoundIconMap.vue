<template>
  <path
    fill="currentColor"
    d="M25 12a6.562 6.562 0 0 0-6.563 6.563c0 2.93 4.29 8.27 5.933 10.209a.821.821 0 0 0 1.26 0c1.643-1.939 5.933-7.28 5.933-10.21A6.562 6.562 0 0 0 25 12zm0 8.75a2.187 2.187 0 1 1 0-4.374 2.187 2.187 0 0 1 0 4.374zm-13.952 2.497A1.667 1.667 0 0 0 10 24.795v13.037c0 .59.595.993 1.143.774l7.19-3.273v-12.14a15.773 15.773 0 0 1-1.106-2.417l-6.18 2.471zM25 30.733a2.489 2.489 0 0 1-1.902-.884A55.015 55.015 0 0 1 20 25.854v9.479l10 3.333V25.854a54.982 54.982 0 0 1-3.098 3.996 2.49 2.49 0 0 1-1.902.883zm13.857-10.34l-7.19 3.274v15l7.285-2.914A1.666 1.666 0 0 0 40 34.205V21.168a.833.833 0 0 0-1.143-.774z"
  />
</template>

<script>
export default {
  name: "RoundIconMap",
};
</script>
