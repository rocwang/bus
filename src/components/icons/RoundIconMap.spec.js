import RoundIconMap from "./RoundIconMap";
import roundIcon from "./__fixtures__/roundIcon.spec";

describe("RoundIconMap.vue", roundIcon(RoundIconMap));
