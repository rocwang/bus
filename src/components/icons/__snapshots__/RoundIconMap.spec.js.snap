// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundIconMap.vue matches the snapshot 1`] = `
<svg
  viewBox="0 0 36 36"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M18 0a18 18 0 1 0 18 17.988C35.994 8.052 27.937 0 18 0"
    fill="#00a3da"
  />
   
  <path
    d="M18.2 8.667a4.812 4.812 0 0 0-4.813 4.812c0 2.149 3.146 6.065 4.35 7.487.245.288.681.288.925 0 1.205-1.422 4.35-5.338 4.35-7.487A4.812 4.812 0 0 0 18.2 8.667zm0 6.416a1.604 1.604 0 1 1 0-3.208 1.604 1.604 0 0 1 0 3.208zM7.968 16.915A1.222 1.222 0 0 0 7.2 18.05v9.56c0 .433.437.728.838.568l5.273-2.4v-8.903a11.567 11.567 0 0 1-.812-1.773l-4.53 1.813zm10.232 5.49a1.825 1.825 0 0 1-1.394-.649 40.344 40.344 0 0 1-2.273-2.93v6.951l7.334 2.445v-9.396a40.32 40.32 0 0 1-2.273 2.93 1.826 1.826 0 0 1-1.394.648zm10.162-7.583l-5.273 2.4v11l5.343-2.137c.464-.185.768-.635.768-1.135v-9.56a.611.611 0 0 0-.838-.568z"
    fill="#da2f2d"
  />
</svg>
`;
